### DISCUSSION TITLE
test on my server

### DISCUSSION THREAD

**Discussion by helpobras** [mysigmail/card#3] [Q&A] [answered]
I found the project interesting. How do I test on my hosting?
what are the pre-requirements of software to install.

**Comment by ibanez32** [comment]
The project seems very promising, but it lack of documentation for quick starting and testing

**Comment by antonreshetov** [comment, marked as answer]
There is nothing to test and run locally yet. Unfortunately, I do not have free time to devote to this project. Currently fully engaged in this project - https://github.com/massCodeIO/massCode. 

But as soon as I find some free time, I will definitely get back to this project!

**Reply by flor-maria** [reply]
Hi Anton! I'm wondering if I could use mysigmail for creating personal e-mails. I noticed I can't create a new account...

**Reply by antonreshetov** [reply]
I disabled the ability to create new accounts.
